tests/translator/test_translator.py

```diff
@@ -271,12 +287,29 @@ def test_transform_success(self, testcase, partition_with_region):
 
         print(json.dumps(output_fragment, indent=2))
 
+        # Run cfn-lint on translator test output files.
+        rules = cfnlint.core.get_rules([], LINT_IGNORE_WARNINGS, [])
+
         # Only update the deployment Logical Id hash in Py3.
         if sys.version_info.major >= 3:
             self._update_logical_id_hash(expected)
             self._update_logical_id_hash(output_fragment)
+            output_template = cfnlint.decode.cfn_json.load(expected_filepath)
+        else: # deprecation warning catching in py2
+            import warnings
+            with warnings.catch_warnings():
+                warnings.filterwarnings("ignore",category=DeprecationWarning)
```

Comment:
CFN lint throws a DeprecationWarning in python2 here for every cfn `node` object created (possibly dozens per template), and Travis truncates the test output due to the sheer volume of warnings generated. This is to make travis (and the `tox` command) play nice with these tests and catch the warning.

Reply:
Travis errors on the output even though the tests pass.

Reply:
See https://travis-ci.org/awslabs/serverless-application-model/jobs/519380376

Reply:
How do we work around this?